ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/Tests.scala

```diff
@@ -6,39 +6,35 @@ package com.daml.ledger.api.testtool
 import java.nio.file.Path
 
 import com.daml.ledger.api.testtool
-import com.daml.ledger.api.testtool.infrastructure.{
-  BenchmarkReporter,
-  LedgerSession,
-  LedgerTestSuite
-}
+import com.daml.ledger.api.testtool.infrastructure.{BenchmarkReporter, LedgerTestSuite}
 import com.daml.ledger.api.testtool.tests._
 import org.slf4j.LoggerFactory
 
 object Tests {
-  type Tests = Map[String, LedgerSession => LedgerTestSuite]
+  type Tests = Seq[LedgerTestSuite]
 
   /**
     * These tests are safe to be run concurrently and are
     * always run by default, unless otherwise specified.
     */
-  val default: Tests = Map(
-    "ActiveContractsServiceIT" -> (new ActiveContractsService(_)),
-    "CommandServiceIT" -> (new CommandService(_)),
-    "CommandSubmissionCompletionIT" -> (new CommandSubmissionCompletion(_)),
-    "CommandDeduplicationIT" -> (new CommandDeduplication(_)),
-    "ContractKeysIT" -> (new ContractKeys(_)),
-    "DivulgenceIT" -> (new Divulgence(_)),
-    "HealthServiceIT" -> (new HealthService(_)),
-    "IdentityIT" -> (new Identity(_)),
-    "LedgerConfigurationServiceIT" -> (new LedgerConfigurationService(_)),
-    "PackageManagementServiceIT" -> (new PackageManagement(_)),
-    "PackageServiceIT" -> (new Packages(_)),
-    "PartyManagementServiceIT" -> (new PartyManagement(_)),
-    "SemanticTests" -> (new SemanticTests(_)),
-    "TransactionServiceIT" -> (new TransactionService(_)),
-    "WitnessesIT" -> (new Witnesses(_)),
-    "WronglyTypedContractIdIT" -> (new WronglyTypedContractId(_)),
-    "ClosedWorldIT" -> (new ClosedWorld(_)),
+  val default: Tests = Seq(
+    new ActiveContractsServiceIT,
+    new ClosedWorldIT,
+    new CommandDeduplicationIT,
+    new CommandServiceIT,
+    new CommandSubmissionCompletionIT,
+    new ContractKeysIT,
+    new DivulgenceIT,
+    new HealthServiceIT,
+    new IdentityIT,
+    new LedgerConfigurationServiceIT,
+    new PackageManagementServiceIT,
+    new PackageServiceIT,
+    new PartyManagementServiceIT,
+    new SemanticTests,
+    new TransactionServiceIT,
+    new WitnessesIT,
+    new WronglyTypedContractIdIT,
   )
```

Comment:
Thank you so much for doing this. It's been bugging me forever.

Reply:
It seems to break CI, though. Well, make it very flaky. It looks like we've somehow ended up in a state where the only ordering that reliably works for our build is the one that was given by the map. Adding an explicit shuffle made the build more likely to succeed, though still very flaky.

Reply:
Throttling to starting no more than 3 tests per second seems to make CI non-flaky. Or at least so much less flaky that I'm not seeing it anymore. We do have close to two hundred tests so that does mean we can't run the full test suite in less than 180s though.